api/src/preDb.ts

```diff
@@ -12,7 +12,7 @@ export async function preDb() {
     "CREATE TABLE IF NOT EXISTS desk (desk_id TEXT PRIMARY KEY, desk_state TEXT)"
   );
```

Comment:
could you rename this file and function to prepareDb? I hope that will be a bit clearer.